### ISSUE TITLE
Pascal VOC mAP equal 0.0

### ISSUE THREAD

**Issue by mtbthebest** [xuannianz/EfficientDet#172]
I tried to run common/eval.py for PASCAL VOC 2007 test but I got a mAP of value equal to 0.0. And I have already the pretrained weights that you provided in the respository homepage. What might be the reason

**Comment by murthy95** [comment]
@mtbthebest I am also facing the same problem. Did you figure it out ?

**Comment by mtbthebest** [comment]
@murthy95 those weights are not correct. Just train the network for 8 epochs by downloading the weights of the efficientnet(imagenet) and freeze the backbone. You will get an mAP of 75% for pascal voc.
`python3 train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32 --steps 1000 pascal datasets/VOC2012`

**Comment by ycdhqzhiai** [comment]
@murthy95 @mtbthebest  I am also facing the same problem. Did you soloved it
